packages/requester-node-http/src/createNodeHttpRequester.ts

```diff
@@ -40,7 +42,7 @@ export function createNodeHttpRequester(): Requester & Destroyable {
 
             resolve({
               status: response.statusCode || 0,
-              content,
+              content: Buffer.concat(contentBuffers).toString(),
```

Comment:
I checked, and this seems to be the same as we had in v3: https://github.com/algolia/algoliasearch-client-javascript/blob/v3.20.3/src/server/builds/node.js#L187

Reply:
Thanks for that info, just finishing up shouldStop issue, and I will jump into this.